aerialist/px4/simulator.py

```diff
@@ -120,8 +130,8 @@ def start(self) -> bool:
                 if self.config.simulator == SimulationConfig.ROS and output.endswith(
                     "INFO  [tone_alarm] home set"
                 ):
-                    logger.info("Avoidance is ready (waiting 5 seconds to wrap up)")
-                    time.sleep(5)
+                    logger.info("Avoidance is ready (waiting 10 seconds to wrap up)")
+                    time.sleep(10)
```

Comment:
we can define it as a variable on top. this depends on how many obstacles are being added, right?

Reply:
I will make thema a variable